### PR TITLE
📖 Add repository guidelines e.g., for project donations

### PR DESCRIPTION
**Pull request by justaugustus**
#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

ref: https://github.com/ossf/scorecard/issues/4073, https://github.com/ossf/scorecard/issues/3204, https://github.com/UlisesGascon/openssf-scorecard-monitor/pull/74

- [ ] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the new behavior (if this is a feature change)?**

- [ ] Tests for the changes have been added (for bug fixes/features)

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note
NONE
```

### CODE REVIEW

**Comment by justaugustus** [comment]
Tagging @UlisesGascon for review on this, as he's actively running through the process in https://github.com/ossf/scorecard/issues/3204, https://github.com/UlisesGascon/openssf-scorecard-monitor/pull/74.

File: docs/repositories.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+New repositories require approval from the OpenSSF Scorecard Steering Committee.
+
+## Requirements
+
+The following requirements apply to all OpenSSF Scorecard repositories:
+
+- Must be identified in the OpenSSF Scorecard project documentation
+- Must reside in the [OpenSSF GitHub organization](https://github.com/ossf)
+- Must utilize the topic [`openssf-scorecard`](https://github.com/topics/openssf-scorecard) (ref: [managing topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics))
```
Comment (spencerschrock) [inline docs/repositories.md:27]: we should mark our other existing repos with this topic

**Review by spencerschrock — COMMENTED** [review]
Ignoring the TODOs LGTM. Is your intent to remove them in this PR?

File: docs/repositories.md (hunk 1 above)
Comment (justaugustus) [reply docs/repositories.md:27]: Done!

**Comment by justaugustus** [comment]
> Ignoring the TODOs LGTM. Is your intent to remove them in this PR?

@spencerschrock — those TODOs need to be addressed in a follow-up. They're items that were present in the Kubernetes guidelines that we should solve for, but aren't blocking to bootstrap the docs.

**Pull request merged**